<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDelete</class>
 <widget class="QDialog" name="DialogDelete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Plugins</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color: #6F7F85;
}

QDialog{
background-color: white;
}

QListWidget::item:selected{
    background: #65B0C9;
}

QPushButton:focus{
	border: none;
	outline: none;
}</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>60</y>
     <width>271</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #BBC7C4;
color: #6F7F85;
font-size: 12pt;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>280</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #BBC7C4;
color: #6F7F85;
font-size: 12pt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: #65B0C9;
background-color: white;
border-radius:6px;
border:1px solid #BBC7C4;
font-family:Ubuntu;
font-size:12pt;
}

QPushButton::hover{
color: white;
background-color: #65B0C9;
border-radius:6px;
border:1px solid #BBC7C4;
font-family:Ubuntu;
font-size:12pt;
}</string>
   </property>
   <property name="text">
    <string>Delete Plugins</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%m%</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
   border: none;
   color: #65B0C9;
}</string>
   </property>
   <property name="text">
    <string>All Plugins</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: #6F7F85;
border: none;
font-size: 12pt;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #65B0C9;</string>
   </property>
   <property name="text">
    <string>Peers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #65B0C9;</string>
   </property>
   <property name="text">
    <string>Loaded Plugins</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
